test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -251,21 +252,21 @@ public void LdaWorkout()
                 .Read(sentimentDataPath);
 
             var est = new WordBagEstimator(env, "text", "bag_of_words").
-                Append(new LdaEstimator(env, "bag_of_words", "topics", 10, advancedSettings: s =>
-                {
-                    s.NumIterations = 10;
-                    s.ResetRandomGenerator = true;
-                }));
+                Append(new LatentDirichletAllocationEstimator(env, "bag_of_words", "topics", 10, numIterations: 10,
+                    resetRandomGenerator: true));
 
             // The following call fails because of the following issue
             // https://github.com/dotnet/machinelearning/issues/969
+            // In this test it manifests because of the WordBagEstimator in the estimator chain
```

Comment:
What would happen if you uncomment this?
I've recently convert WordBag to estimator so it should work properly.

Reply:
If I uncomment this, I get an error 

<snip>
Schema mismatch for input column 'text': expected Text, got R4
Parameter name: inputSchema
</snip>

The `WordBagWorkout` test still comments out the `TestEstimatorCore` (line #186 above) .   

---
In reply to: [235166598](https://github.com/dotnet/machinelearning/pull/1410#discussion_r235166598) [](ancestors = 235166598)